system/doc/reference_manual/features.xml

```diff
@@ -0,0 +1,204 @@
+      default.</item>
+      <tag>Approved</tag>
+      <item>The feature has been finalised and is now part of OTP.  By
+      default it is enabled, but can be disabled.</item>
+      <tag>Permanent</tag>
+      <item>The feature is now a permanent part of OTP.  It can no
+      longer be disabled.</item>
+      <tag>Rejected</tag>
+      <item>The feature never reached the approved state and will not
+      be part of OTP.  It cannot be enabled.</item>
+    </taglist>
+    <p>
+      After leaving the experimental state, a feature can enter any of
+      the other three states, and if the next state is approved, the
+      feature will eventually end up in the permanent state.  A feature
+      can change state only in connection with a release.
+    </p>
+    <p>
+      A feature may be in the approved state for several releases.
+    </p>
+    <table>
+      <row>
+        <cell>State</cell>
+        <cell>Default</cell>
+        <cell>Controllable</cell>
+        <cell>Available</cell>
+      </row>
+      <row>
+        <cell>Experimental</cell>
+        <cell>disabled</cell>
+        <cell>yes</cell>
+        <cell>yes</cell>
+      </row>
+      <row>
+        <cell>Approved</cell>
+        <cell>enabled</cell>
+        <cell>yes</cell>
+        <cell>yes</cell>
+      </row>
+      <row>
+        <cell>Permanent</cell>
+        <cell>enabled</cell>
+        <cell>no</cell>
+        <cell>yes</cell>
+      </row>
+      <row>
+        <cell>Rejected</cell>
+        <cell>disabled</cell>
+        <cell>no</cell>
+        <cell>no</cell>
+      </row>
+      <tcaption>Feature States</tcaption>
+    </table>
+    <list>
+      <item>Being controllable means the possibility to enable or
+      disable the feature by means of compiler options and directives
+      in the file being compiled.</item>
+      <item>Being available can be seen using the
+      <c>FEATURE_AVAILABLE</c> macro.</item>
+    </list>
+  </section>
+
+  <section>
+    <title>Enabling and Disabling Features</title>
+    <p>To use a feature that is in the experimental state, it has to
+    be enabled during compilation.  This can be done in a number of
+    different ways:
+    </p>
+    <taglist>
+      <tag>Options to <c>erlc</c></tag>
+      <item>Options <seecom
+      marker="erts:erlc#enable-feature"><c>-enable-feature</c></seecom>
+      and <seecom
+      marker="erts:erlc#disable-feature"><c>-disable-feature</c></seecom>
+      can be used to enable or disable individal features.</item>
+      <tag>Compiler options</tag>
+      <item>The compiler option <seeerl
+      marker="compiler:compile#feature-option"><c>{feature,
+      &lt;feature&gt;, enable|disable}</c></seeerl> can be used either
+      as a <c>+&lt;term&gt;</c> option to <c>erlc</c>, in the options
+      argument to function in the <c>compile</c> module, or in the
+      <c>-compile(..)</c> attribute in a file.</item>
+      <tag>The feature directive</tag>
```

Comment:
```suggestion
      <tag>The feature attribute</tag>
```

Reply:
Attributes have only one argument, while the `-feature` directive has two, although the documentation mentions other constructs, e.g., `-record(..)` sharing the same syntax as attributes.  The documentation on the preprocessor mentions directives.  I see this more akin to a directive since it has a direct affect will/can affect the parsing of the file.

Reply:
Yes, now I see it. Use directive.